main/res/values/strings.xml

```diff
@@ -2029,27 +2029,27 @@
     <string name="next">Next</string>
     <string name="finish">Finish</string>
     <string name="wizard_welcome_title">Welcome to c:geo</string>
-    <string name="wizard_intro">It looks like this is the first time c:geo is used on this device. Shall I help you with configuration? It\'s just a few steps.</string>
-    <string name="wizard_intro2">Welcome back to the configuration wizard. It\'s just a few steps.</string>
+    <string name="wizard_intro">Just a few steps are needed to configure c:geo and get you going. Shall I help you with that?</string>
+    <string name="wizard_intro2">Welcome back to the configuration wizard. Your c:geo setup is not yet finished, shall I help you to complete it and get you going? It\'s just a few steps.</string
     <string name="wizard_outro_ok">Thanks for configuring c:geo.\nHave fun!</string>
     <string name="wizard_outro_error">Incomplete configuration\nNot all parts of c:geo will work as expected!</string>
     <string name="wizard_permissions_title">Set app permissions</string>
-    <string name="wizard_permissions_intro">c:geo needs certain permissions to function correctly. I will explain them in the next steps.\nWithout these permissions you will not be able to use c:geo.</string>
-    <string name="wizard_platforms_title">Configure platforms</string>
-    <string name="wizard_platforms_intro">To download caches you need an account for at least one of the geocaching platforms supported by c:geo</string>
+    <string name="wizard_permissions_intro">c:geo needs certain permissions on your device to work correctly.\nI will explain more details in the next steps.</string>
+    <string name="wizard_platforms_title">Configure services</string>
```

Comment:
Hmm, we had the problem before that the dialog presents "services" as acronym for services, where you manage caches (geocaching.com, opencaching*. , ...) and services for additional information (rating, ...). To distinguish the use cases the term platform was used for the cache-provider. 
I think we have to use a clean wording so that new user can this distinguish.
So I would stay with platform, if this is correct for english native speeker, or at least use the term "geocaching service" all the time.

Reply:
Maybe "Configure geocaching services" or "Configure geocaching platforms" to make the meaning a bit more clear...

Reply:
I took a look at the installation wizard, and you are able to configure both, platforms as well as further services. So "Services" is the correct wording...

Reply:
I had intentionally chosen the wording "platforms", as the focus in this step should be to select a geocaching platform, like gc.com or oc.de, not a specialized service like gcvote.com

I had a trade-off to make here: gc.com is by far the most widely used platform (especially for geocaching beginners, which this wizard is mostly intended for), so I put in an extra button for it, linked directly to the specific settings page. The other platforms are much less common and more regional, so I did not want to put an extra button on the page for every other platform, but just linked to our general services settings page - which indeed lists both services and platforms. Maybe we need to split-up our settings pages accordingly into "platforms" and "other services"?

Reply:
IMHO a geocaching platform is also a service. In our settings we use that wording too.

Reply:
Thats why I also ended up with service here. If I look into c:geo Settings the first item is "Services".

Reply:
No change done here.